internal/support/notifications/rest_subscription.go

```diff
@@ -58,6 +58,13 @@ func subscriptionHandler(w http.ResponseWriter, r *http.Request) {
 		dec := json.NewDecoder(r.Body)
 		err := dec.Decode(&s)
 
+		// validate if email addresses contain CRLF pair
+		if err = validateEmailAddresses(s); err != nil {
```

Comment:
Should we add some error type checking here and return `http.StatusBadRequest` if the error is of type `ErrMailAddrContainsCRLF` otherwise return a `http.StatusInternalServerError`?

Reply:
Yes, done.